all-imports.html

```diff
@@ -1,3 +1,3 @@
 <!-- all-imports.html -->
 <link rel="import" href="i18n-msg.html">
-<link rel="import" href="i18n-page-example.html">
+<link rel="import" href="demo/i18n-msg-behavior-example.html">
```

Comment:
Out of curiosity, wouldn't it make more sense to import the i18n-msg-behavior.html instead of the example element?

Maybe I'm missing the purpose of the all-imports.html ?